.github/workflows/check-outdated-trusted-root.yml

```
@@ -7,6 +7,9 @@ on:
 jobs:
   check:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
```

```suggestion
      # issues:write is required to create issues.
      issues: write
```
The other `write` permissions have a short explanatory comment. Here it's a little tautological, but for consistency 😄